types/debug/index.d.ts

```diff
@@ -25,6 +30,7 @@ declare namespace debug {
         log: (...args: any[]) => any;
         selectColor: (namespace: string) => string | number;
         humanize: typeof import('ms');
+        inspectOpts?: InspectOptions;
```

Comment:
These options are only available in `node`, so they are made optional here since they may not exist if running in the browser